andrada12/assignment4/WorldMap.java

```diff
@@ -0,0 +1,135 @@
+
+import java.util.ArrayList;
+import java.util.TreeSet;
+
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+
+/**
+ *
+ * @author andrada
+ */
+
+ class MapPart{
```

Comment:
It would be nice to have some javadoc explaining these, as "type" could mean almost anything.